python-package/xgboost/sklearn.py

```diff
@@ -497,7 +503,7 @@ def fit(self, X, y, sample_weight=None, eval_set=None, eval_metric=None,
                               evals=evals,
                               early_stopping_rounds=early_stopping_rounds,
                               evals_result=evals_result, obj=obj, feval=feval,
-                              verbose_eval=verbose)
+                              verbose_eval=verbose, xgb_model=None)
```

Comment:
Hi.  Why was xgb_model=None set.  This should have been like above xgb_model=xgb_model.

Reply:
This is an overlook. Will include the fix when h2oai PR into dmlc. Thanks @pseudotensor!